src/Lesson3/HomeTask5.java

```diff
@@ -0,0 +1,13 @@
+//5. Given 2 strings, a and b. Create a string c of the form short+long+short, with the shorter string on the outside and the longer string on the inside.
+//
+//   Task: a = "hop", b = "hello"
+package Lesson3;
+
+public class HomeTask5 {
+    public static void main(String[] args){
+        String a = "hop";
+        String b = "hello";
+
+System.out.println(a+ " " + b + " "+ a);
```

Comment:
In this task you should write a code with if/else for checking length of incoming string. If one string longer that another then it should be at the middle of the third string.